### PR TITLE
Feature/ffmpeg layouts

### PR DESCRIPTION
**Pull request by Euklios**
Add support for the `ffmpeg -layouts` info command

### CODE REVIEW

File: src/main/java/net/bramp/ffmpeg/info/Layout.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package net.bramp.ffmpeg.info;
+
+public interface Layout {
```
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/info/Layout.java:3]: Should this be called ChannelLayout?

**Comment by bramp** [comment]
This change looks good. Happy to accept, but it seems a conflict is stopping me from rebase/merging

File: src/main/java/net/bramp/ffmpeg/info/Layout.java (hunk 1 above)
Comment (Euklios) [reply src/main/java/net/bramp/ffmpeg/info/Layout.java:3]: I used Layout because it's the name used by ffmpeg. But I agree, your suggestion is better.

**Comment by Euklios** [comment]
@bramp Merged in master and renamed.
I like the new CI.

**Pull request merged**